### PR TITLE
Nits from various issues.

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

**Review by csperkins — APPROVED** [review]

File: draft-ietf-taps-transport-security.md [hunk 1]
```diff
@@ -328,7 +328,7 @@ DTLS is modified from TLS to operate with the possibility of packet loss, reorde
...
 
-DTLS relies on unique UDP 4-tuples to allow peers with multiple DTLS connections between them to demultiplex connections, constraining protocol design slightly more than UDP: application-layer demultiplexing over the same 4-tuple is not possible without trial decryption as all application-layer data is encrypted to a connection-specific cryptographic context. Starting with DTLS 1.3 {{I-D.ietf-tls-dtls13}}, a connection identifier extension to permit multiplexing of independent connections over the same 4-tuple is available {{I-D.ietf-tls-dtls-connection-id}}.
+DTLS relies on unique UDP 4-tuples to allow peers with multiple DTLS connections between them to demultiplex connections, constraining protocol design slightly more than UDP: application-layer demultiplexing over the same 4-tuple is not possible without trial decryption as all application-layer data is encrypted to a connection-specific cryptographic context. {{I-D.ietf-tls-dtls-connection-id}} specifies a connection identifier extension for DTLS 1.2 and 1.3 {{I-D.ietf-tls-dtls13} that permits multiplexing of independent connections over the same 4-tuple.
```
Comment (squarooticus) [inline draft-ietf-taps-transport-security.md]: This is a bit awkward. "Not possible" isn't true if the extension is used. Maybe reword to say "As all application-layer data is encrypted, demultiplexing over the same 4-tuple requires the use of a connection identifier extension {{I-D.ietf-tls-dtls-connection-id}} to permit identification of the correct connection-specific cryptographic context without the use of trial decryption."

**Review by squarooticus — CHANGES_REQUESTED** [review]

File: draft-ietf-taps-transport-security.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-taps-transport-security.md]: Yeah, it is awkward. I pushed a variation of your proposed text. Please have a look!

**Review by squarooticus — APPROVED** [review]
